src/commands/apps/info.js

```diff
@@ -8,6 +8,8 @@ function * run (context, heroku) {
   const util = require('util')
   const S = require('string')
   const countBy = require('lodash.countby')
+  //const filter = require('lodash.filter')
+  const _ = require('lodash');
```

Comment:
this causes a big performance hit in the CLI if you require at the top of the file like this. I would rewrite it so you can use lodash functions by themselves rather than requiring everything. Unfortunately this means you won't be able to use chain though. I can give you a hand optimizing it if you like.